django_celery_results/urls.py

```diff
@@ -32,23 +34,52 @@ def to_url(self, value):
 
 urlpatterns = [
     path(
-        '<task_pattern:task_id>/done/',
+        'task/done/<task_pattern:task_id>/',
         views.is_task_successful,
         name='celery-is_task_successful'
     ),
     path(
-        '<task_pattern:task_id>/status/',
+        'task/status/<task_pattern:task_id>/',
         views.task_status,
         name='celery-task_status'
     ),
     path(
-        '<task_pattern:group_id>/group/done/',
+        'group/done/<task_pattern:group_id>/',
         views.is_group_successful,
         name='celery-is_group_successful'
     ),
     path(
-        '<task_patern:group_id>/group/status/',
+        'group/status/<task_pattern:group_id> /',
         views.group_status,
         name='celery-group_status'
     ),
 ]
+
+if getattr(settings, 'DJANGO_CELERY_RESULTS_ID_FIRST_URLS', True):
+    warnings.warn(
```

Comment:
Good point to say when it will be removed. 
As project has no strict releases you can say something like
`Will be removed in 2022`
and remove this part in first or any version in 2022. It will give ~6 month to developers.